src/foam/u2/borders/ScrollBorder.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'ScrollBorder',
+  extends: 'foam.u2.Element',
+
+  css: `
+    ^ {
+      display: flex;
+      flex-direction: column;
+    }
+    ^reverse {
+      display: flex;
+      flex-direction: column-reverse;
+      justify-content: flex-end;
+    }
+    ^::before {
+      display: block;
+      opacity: 0;
+      height: 7px;
+      min-height: 7px;
+      width: 100%;
+      content: '';
+      z-index: 10;
+      background: linear-gradient(to bottom, #DADDE2 0, #ffffff00 7px, #ffffff00 100%);
+      top: 0;
+      position: sticky;
+    }
+    ^::after {
+      display: block;
+      opacity: 0;
+      height: 7px;
+      min-height: 7px;
+      width: 100%;
+      content: '';
+      z-index: 10;
+      background: linear-gradient(to top, #DADDE2 0, #ffffff00 7px, #ffffff00 100%);
+      bottom: 0;
+      position: sticky;
+    }
+    ^topShadow::before {
+      opacity: 1;
+    }
+    ^botShadow::after {
+      opacity: 1;
+    }
+    ^::-webkit-scrollbar {
+      width: 0;  /* Remove scrollbar space */
```

Comment:
Not so sure about this, wouldnt this hide the scrollbar for everyone?

Reply:
Yeah I meant to remove that, just wanted to see what it looked like